src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/FeatureToggleService.java

```diff
@@ -86,6 +88,10 @@ public boolean isCaseFileViewEnabled() {
         return isFeatureEnabled(CASE_FILE_VIEW_ENABLED);
     }
 
+    public boolean isExpressPilotEnabled() {
+        return isFeatureEnabled(Features.EXPRESS_PILOT_ENABLED);
+    }
+
```

Comment:
I think you took the log out again.  So you can take out the logger annotation too.

Reply:
Import?